app/views/mentoring/external/show.html.haml

```diff
@@ -11,7 +12,10 @@
         Some of our best mentors worried too.
 
       - if user_signed_in?
-        = ReactComponents::Mentoring::TryMentoringButton.new.to_s
+        - if current_user.reputation >= 20
+          = ReactComponents::Mentoring::TryMentoringButton.new.to_s
```

Comment:
```suggestion
          = ReactComponents::Mentoring::TryMentoringButton.new
```

You shouldn't need `to_s`. It should get automatically called (I know this was in the bit before - maybe you can check for other places to remove it to?)

Reply:
ok!